docker/post_github_comment.py

```diff
@@ -46,62 +47,52 @@ def get_docker_image_size(docker_image, is_contribution: bool = False) -> str:
     return size
 
 
-def convert_docker_image_tar(docker_image: str) -> Path:
+def convert_docker_image_tar(docker_image: str) -> str:
     """
-    Helper function to convert the Docker image to valid path on CircleCI worker. For example:
-    `devdemisto/bottle2:1.0.0.89478.tar.gz` -> `devdemisto_bottle2:1.0.0.89478.tar.gz`.
+    Helper function to convert the Docker image to a valid path.
+    For example, `devdemisto/bottle2:1.0.0.89478.tar.gz` -> `devdemisto_bottle2:1.0.0.89478.tar.gz`.
```

Comment:
What if the image contains a `_`?
Maybe use `__` (2 underscores) as the delimiter?

Reply:
fixed